### DISCUSSION TITLE
Failed to get copilot token

### DISCUSSION THREAD

**Discussion by Manvendra-div** [community/community#9922] [Copilot Conversations]
![ArcoLinux_2022-01-10_17-56-47](https://user-images.githubusercontent.com/67854646/148765836-0aabd514-3e73-469f-a64c-d971776feba9.png)
**How to fix this,**
_Help very urgent_ **kidding**🤣
but tell me how to fix

**Comment by davecheney** [comment]
This error there was an error connecting to api.github.com to request a token to use the Copilot service. Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by Manvendra-div** [reply]
![ArcoLinux_2022-01-11_10-47-18](https://user-images.githubusercontent.com/67854646/148885599-3b8d6cd1-6297-4c9f-908b-9dd2479c2ccd.png)
![ArcoLinux_2022-01-11_10-48-20](https://user-images.githubusercontent.com/67854646/148885607-d0d5aa26-418b-4221-9358-bd6dbe2a0b0c.png)
yes **done**

**Reply by davecheney** [reply]
Hello, I'm sorry you are having difficulties. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by Manvendra-div** [reply]
I am using Arcolinux Arch based distro.

**Reply by davecheney** [reply]
Is there something about the network you're on, your employer, or your ISP that may be intercepting your SSL traffic?

**Reply by Manvendra-div** [reply]
i am using **Vi(Vodafone Idea)** ISP and i am not using any firewall software

**Reply by davecheney** [reply]
You mentioned you were using arch linux, is it possible you have installed code, or vscodium rather than vscode? copilot only supports vscode, not their unbranded variants often packaged by linux distributions.

**Reply by Manvendra-div** [reply]
No problem was not like that I was using Official *Microsoft Visual Studio Code*,the real problem was the firewall was enabled by default in my current distro *Arcolinux* so for disable firewall i used to install package called gufw,
So it's fixed btw😁
Thanks you all bro/sister

**Reply by geofannyher** [reply]
me to

**Reply by Gustavo10Destroyer** [reply]
https://code.visualstudio.com/docs/editor/settings-sync#_troubleshooting-keychain-issues

**Reply by Gustavo10Destroyer** [reply]
You can try this for Debian:
sudo apt install gnome-keyring

And this for Arch:
sudo pacman -S gnome-keyring

(Works for me)
(Sorry, my english is bad)
